src/router/index.ts

```diff
@@ -2,7 +2,7 @@ import { createRouter, createWebHistory, RouteRecordRaw } from "vue-router";
 import Home from "../views/Home.vue";
 import Profile from "../views/Profile.vue";
 import ProfileUpload from "../components/profile/profile-upload/ProfileUpload.vue";
-import ProfileDetail from "../components/profile/profile-detail/ProfileDetail.vue";
+import ProfileDetail from "@/views/profile/ProfileDetail.vue";
```

Comment:
Let's name in similar way as upload on Idowu PR, so `ProfileDetail.vue` -> `Detail.vue`